cli.ts

```diff
@@ -80,6 +81,11 @@ async function cli(args: ParsedArgs) {
     return
   }
 
+  if (args.version) {
+    console.log(version);
+    process.exit(0);
+  }
```

Comment:
make sure to run prettier

```suggestion
    console.log(version)
    process.exit(0)
  }
```